svthalia/concrexit
# Update tcp info

### Summary
Replaced Tom with myself as TCP.
Updated Lisa's picture.

### How to test
Go to the "student well-being" page (under Education).
See if the picture, the name, the hover-over email address and the on-click email creation work as intended.
